msg/Entity.msg

```diff
@@ -0,0 +1,5 @@
+# Entity ground truth state in the simulation
+
+string name                                 # Entity's scoped, unique name (including namespace)
+string description                          # Optional: additional information about entity.
+EntityState state                           # Entity current state
```

Comment:
Consider the removal of EntityState - or having two variants of this message - the other one without a state.
Rationale: 
This potentially increases the amount of computation needed for handling GetEntities. it also increases the size of the message by roughly 100B. Depending on the use case it might not be required by a caller so the interface should not force it